spec/statesman/adapters/shared_examples.rb

```diff
@@ -11,13 +11,13 @@
 #   history:          Returns the full transition history
 #   last:             Returns the latest transition history item
 #
-shared_examples_for "an adapter" do |adapter_class, transition_class|
+shared_examples_for "an adapter" do |adapter_class, transition_class, association_name = nil|
```

Comment:
Can this become `|adapter_class, transition_class, options = {}|` instead so it's the same as the initializer? This line could get very long if more options get added otherwise!